blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,224 @@
+---
+title: The workspaces of Sourcegraph
+author: Rob Rhyne
+authorUrl: https://twitter.com/robrhyne
+publishDate: 2021-02-10T18:00+02:00
+tags: [blog, remote, design]
+slug: workspaces-of-sourcegraph
+published: true
+---
+
+The Sourcegraph design team is exploring how to improve the display of code throughout the application. As part of this exploration, we are reviewing trends in typography, color schemes, and UI complexity in the places our engineers view code the most… their editors.  While our research is far from complete, we thought you might find a breakdown of the visual characteristics of where our team writes code of interest. 
+
+## Light/dark mode and contrast
+Three quarters of our engineers preferred a dark theme over a light theme.  Curiously, the majority of dark mode users chose a low contrast between code and background. Backgrounds colors averaged 85% black while the  primary code color to background contrast ratio was around 9 to 1. 
+
+Conversely, all light mode users at Sourcegraph prefer the maximum contrast with 100% white backgrounds and a black primary code color. 
+
+![Representative contrast](https://sourcegraphstatic.com/blog/workspaces-of-sourcegraph/workspace-contrast.png)
+*Contrast representative of typical dark and light modes.*
+
+## Information density
+There are clearly two camps… those who use an IDE (mainly VS Code) and terminal based editors such as neovim/vim/emacs users. Editor users had around 96% of editor screen real estate dedicated to code. IDE users had from 80 to 90% of their screens is dedicated to code or the terminal, with the rest occupied by file trees and other UI such as menus and tabs. 
+
+## Fonts are commonly customized
+Over half of our engineers customize their programing font.  Among these users, [FiraCode](https://github.com/tonsky/FiraCode) was the most popular choice (those [programming ligatures](https://github.com/tonsky/FiraCode#whats-in-the-box) are amazing) followed by  [JetBrains Mono](https://www.jetbrains.com/lp/mono/). Other fonts include  [MonoStroom](https://github.com/Strum355/MonoStroom) , [Cascadia Code](https://github.com/microsoft/cascadia-code)),  and [Hack](https://sourcefoundry.org/hack/).
+
+## What we learned
+While these results are not statistically significant, one trend does stand out... developers will customize their environments to better suit their style of development. 
```

Comment:
Why? I think we can pull this learning together with the intro do the workspaces. For example, 

"While these results are not statistically significant...suit their style of development. Personalizing your development environment does x,y,z. To show off just how different each developer's workspace can be, we pulled together X# of individual setups from Sourcegraphers. 

Move up the prompt and remove the header:
`What editors, themes and fonts do you use? DM or tweet us at [@srcgraph](https://twitter.com/srcgraph) to share your thoughts and images. /Make sure to not include sensitive information in any screens you share!/`

Add a header before the workspaces start: `## Workspaces of Sourcegraph`

Reply:
I think the below captures this request?